pkg/uploader/uploader.go

```diff
@@ -50,10 +52,11 @@ func NewUploaderWithClient(log *libbuildpack.Logger, client AMSClient) Uploader
 }
 
 func (up *uploader) Upload(rootDir string, dstURL string) error {
+	up.root = rootDir
 	up.log.Info("creating policy archive..")
-	buf, err := createArchive(up.log, rootDir)
+	buf, err := up.createArchive(up.log, rootDir)
 	if err != nil {
-		return fmt.Errorf("could not create policy bundle.tar.gz: %w", err)
+		return fmt.Errorf("could not create policy dcl.tar.gz: %w", err)
```

Comment:
nit: should we capitalize DCL (everywhere)?

Reply:
There is a lint that error messages should not start capitalized. Error messages can start with dcl. So I'm insecure how to proceed

Reply:
From the [official wiki](https://github.com/golang/go/wiki/CodeReviewComments#error-strings):
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) 

-> If error messages start with`DCL` it is fine to capitalize it.

Reply:
fixed